src/compinents/calculator.js

```diff
@@ -0,0 +1,27 @@
+import './calculator.css';
+
+const Calculator = () => (
+  <div className="btn_wrapper btn">
+    <div className="display">0</div>
```

Comment:
Thank you @BenMukebo   for your comment, I fixed everything but since I requested the first review my project was running without any error, kindly try to check from your side. 😉 💯